### PR TITLE
Fix for #6469

### PR DESCRIPTION
**Pull request by mklement0**
# PR Summary

Amends and corrects the instructions for installing / updating the PackageManagement and PowerShellGet modules on computers running Windows PowerShell v3 and v4.

* Clarifications re `Save-Module` use and what "PackageManagement Preview" refers to.
* Streamlining of the commands to provide a higher degree of automation.
* Added required command for v3 to ensure that the modules are auto-loaded.
* Minor copy-editing.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [x] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated (N/A)
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/gallery/installing-psget.md [hunk 1]
```diff
@@ -56,32 +54,53 @@ For more information, see
...
+```
 
 #### Computers with the PackageManagement Preview installed
 
-1. From a PowerShell session, use `Save-Module` to save the modules to a local directory.
+> [!NOTE] 
+> PackageManagement Preview was a downloadable component that made PowerShellGet available to PowerShell versions 3 and 4, but it is no longer available.
+> To test if it was installed on a given computer, run `Get-Module -ListAvailable PowerShellGet`.
+
+1. From a PowerShell session, use `Save-Module` to save the use `Save-Module` to download the current version of **PowerShellGet**. Two folders are downloaded: **PowerShellGet** and **PackageManagement**. Each folder contains a subfolder with a version number.
```
[the remark this answers is not in the document]
Comment (mklement0) [reply reference/docs-conceptual/gallery/installing-psget.md]: The latter - thanks for catching that. Fixed.

File: reference/docs-conceptual/gallery/installing-psget.md [hunk 2]
```diff
@@ -56,32 +54,53 @@ For more information, see
...
 > [!NOTE]
 > PowerShell 3.0 and PowerShell 4.0 only supported one version of a module. Starting in PowerShell
-> 5.0, modules are installed in `<modulename>\<version>`. This allowed you to install
+> 5.0, modules are installed in `<modulename>\<version>`. This allows you to install
 > multiple versions side-by-side. After downloading the module using `Save-Module` you must copy the
-> files from the `<modulename>\<version>` to the `<modulename>` folder on the destination machine.
+> files from the `<modulename>\<version>` to the `<modulename>` folder on the destination machine,
+> as shown in the instructions below.
+
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (mklement0) [reply reference/docs-conceptual/gallery/installing-psget.md]: I've turned the note into a separate section entitled "Preparatory Step on computers running PowerShell 3.0" - I hope that makes it clear.

**Comment by chasewilson** [comment]
Looks good to me. Wait for final build and I'll merge

**Comment by chasewilson** [comment]
@mklement0 Thanks for the contribution, always very helpful and informative to hear from you and really great to get a PR from you :) Have a great one!

**Comment by mklement0** [comment]
My pleasure, @chasewilson, and thanks for the nice feedback.

**Pull request merged**